doc/releases/migration-guide-3.7.rst

```diff
@@ -714,6 +687,33 @@ Watchdog
 Bluetooth
 *********
 
+Bluetooth HCI
```

Comment:
It may be nice to add an anchor for this section you can refer in the release notes?